File: src/1318-Minimum-Flips-to-Make-a-OR-b-Equal-to-c/1318.py
class Solution:
    def minFlips(self, a: int, b: int, c: int) -> int:
        res = 0
        while c or a or b:
            if c & 1:
                res += 0 if (a & 1) or (b & 1) else 1
            else:
                res += (a & 1) + (b & 1)
            c >>= 1
            a >>= 1
            b >>= 1
        return res

File: src/1318-Minimum-Flips-to-Make-a-OR-b-Equal-to-c/1318.go
func minFlips(a int, b int, c int) int {
    res := 0
    for a > 0 || b > 0 || c > 0 {
        if c & 1 == 1 {
            if a & 1 == 0 && b & 1 == 0 {
                res++
            }
        } else {
            res += (a & 1) + (b & 1)
        }
        a >>= 1
        b >>= 1
        c >>= 1
    }
    return res
}